<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>582</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushProgressBar">
         <property name="text">
          <string>ProgressBar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushSpinBox">
         <property name="text">
          <string>SpinBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushSlider">
         <property name="text">
          <string>Slider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushRadioButton">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushMessDialog">
         <property name="text">
          <string>Message Dialog</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushListWidget">
         <property name="text">
          <string>ListWidget</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushLCD">
         <property name="text">
          <string>LCD Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushInputDialog">
         <property name="text">
          <string>Input Dialog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushComboBox">
         <property name="text">
          <string>ComboBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushCheckBox">
         <property name="text">
          <string>Check Box</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCalendar">
         <property name="text">
          <string>Calendar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushLineEdit">
         <property name="text">
          <string>LineEdit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushTable">
         <property name="text">
          <string>Table</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushAnimation">
         <property name="text">
          <string>Animation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushAnimation_2">
         <property name="text">
          <string>Animation 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushChart">
         <property name="text">
          <string>Chart</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushClock">
         <property name="text">
          <string>Clock</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushDragDrop">
         <property name="text">
          <string>DragDrop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushDiagrams">
         <property name="text">
          <string>Diagrams</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushMusicPlayer">
         <property name="text">
          <string>Music Player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushVideoPlayer">
         <property name="text">
          <string>Video Player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushWebBrowser">
         <property name="text">
          <string>Web Browser</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
